applications/launchpad/gui-react/src/components/Onboarding/OnboardingMessages/LastStepsMessages.tsx

```diff
@@ -18,12 +21,46 @@ import {
 import ProgressBar from '../../ProgressBar'
 import { TBotMessage, TBotMessageHOCProps } from '../../TBot/TBotPrompt/types'
 import { useTheme } from 'styled-components'
+import { SyncType, useBaseNodeSync } from '../../../useBaseNodeSync'
+import Loading from '../../Loading'
 
 /**
- * @TODO - how the Blockchain synchronization should to work?
+ * Convert from seconds into hours-minutes-seconds
+ * @param {number} time - time in seconds
+ * @param {string}
  */
+const humanizeTime = (time: number) => {
```

Comment:
we already have humanizeTime in utils/Format, maybe we should use that one?

Reply:
Good point! It's not the same function, because in `humanizeTime` converts to `00:00:00`, while this function converts to `00 h 00 min 00 s`, but I move it to the Format utils and named `humanizeEstimatedTime`.